addons/html_editor/static/src/playground.js

```diff
@@ -1,14 +1,21 @@
 /** @odoo-module */
-import { Component } from "@odoo/owl";
+import { Component, useState } from "@odoo/owl";
 import { registry } from "@web/core/registry";
 import { useWysiwyg } from "./editor/wysiwyg";
 
 export class Playground extends Component {
     static template = "html_editor.Playground";
+    static components = { };
     static props = ["*"];
 
     setup() {
         this.editor = useWysiwyg("html");
+        this.state = useState({ showWysiwyg: false });
+        this.constructor.components.Wysiwyg = odoo.loader.modules.get("@web_editor/js/wysiwyg/wysiwyg").Wysiwyg;
```

Comment:
Isn't this like a lazy loading that always happen? (always is our playground, at least)
Or is the goal here to avoid loading such assests when running the test suite?